AdobeStockAsset/Model/Client.php

```diff
@@ -92,4 +100,9 @@ private function getClient()
             $this->config->getTargetEnvironment()
         );
     }
+
+    private function locale(): string
```

Comment:
Is there a reason to use a private method instead of direct usage of the dependency?

Reply:
First I extracted the locale value to a method in order to express the concept. And then I chose the LocaleResolver to provide the expected value. But now I think that changing the name of the collaborator I will achieve the same avoiding one level of indirection :)